src/components/BreedsList.vue

```diff
@@ -155,33 +165,36 @@ export default {
         });
       }
     },
-    toggleCard(breed, refLi) {
+    toggleCard(breed) {
+      console.log(breed.isActive, breed.name, this.$route.name);
       breed.isActive = !breed.isActive;
-      refLi.scrollIntoView({
-        behavior: "smooth",
-        block: "center",
-      });
+
       this.displayedBreeds.forEach((item) => {
-        if (item.id !== breed.id) {
+        if (item.name !== breed.name) {
           item.isActive = false;
         }
       });
+      if (!breed.isActive) {
+        console.log("push");
+        this.$router.push({
+          name: "breeds",
+        });
+      }
     },
-    showBreedDetailsOnRouteParam(breedName) {
-      if (breedName && this.$refs.resultListContainer) {
-        this.displayedBreeds.forEach((breed, id) => {
-          if (breed.name.toLowerCase() === breedName.toLowerCase()) {
-            breed.isActive = true;
+    scrollToBreedCard(breedName) {
+      const breedNameParam = breedName.params.breedName;
+      console.log("scrollToBreedCard method");
+      console.log("breedName:", breedNameParam);
+
+      if (breedNameParam) {
+        this.displayedBreeds.filter((breed, id) => {
+          if (breedNameParam.toLowerCase() === breed.name.toLowerCase()) {
```

Comment:
Naletel sem na težavo, in sicer, ko klikam po cardsih, mi jih načeloma vse odpre in zapre, tudi scrolla jih na sredino, tudi routsi delujejo, vendar pa nekako ne morem ugotoviti težave, ko grem nazaj in naprej z browser gumbom. Mogoče se tudi jaz narobe predstavljam, vendar sem na različnih straneh poizkusil in stvar deluje. Torej želim, da ko odpiram in zapiram cardse in s tem menjam routse, da ko grem nazaj ,da mi odpre tistega prejšnjega in ga scrolla na sredino, in tudi če grem čisto nazaj na prvega odprtega, mi vsakega scrolla na sredino in zapre vse ostale. Poizkušal sem z watchers, z updated hooks, vendar nekako ne pridem naprej nazaj. Gledam že to par dni in se mi zdi, da sem čisto blizu rešitve, ampak vedno neka stvar ne deluje. Če lahko malo stestiraš. Aja za search routsi še ne delujejo.
Hvala :)